app/exporters/patients_exporter.rb

```diff
@@ -16,10 +30,10 @@ def self.csv(patients)
           :phone_numbers,
           :address,
           :medical_history,
-          :prescription_drugs,
+          :current_prescription_drugs,
           :latest_bp_passports,
           { latest_scheduled_appointments: :facility },
-          { latest_blood_pressures: :facility },
+          { latest_blood_pressures: [:facility, {encounter: :blood_sugars}] },
```

Comment:
> Populates blood sugar data into the new columns if and only if the patient had a blood sugar reading on the same day as their latest BP

So if a patient has only a Blood sugar recorded, we will not add a row in the line list? Could you point me to some context around this, a bit out of the loop.